src/shared/crypto/bitcoin/bitcoin.utils.ts

```diff
@@ -64,35 +65,33 @@ export function decodeBitcoinTx(tx: string) {
 export function getAddressFromOutScript(script: Uint8Array, bitcoinNetwork: BtcSignerNetwork) {
   const outputScript = btc.OutScript.decode(script);
 
-  if (outputScript.type === 'pk' || outputScript.type === 'tr') {
-    return btc.Address(bitcoinNetwork).encode({
-      type: outputScript.type,
-      pubkey: outputScript.pubkey,
-    });
+  switch (outputScript.type) {
+    case 'pkh' || 'sh' || 'wpkh' || 'wsh':
```

Comment:
This will likely get closed, but reminder this is written incorrectly and needs to be fixed.